src/controller/CHIPDeviceController.cpp

```diff
@@ -556,24 +556,31 @@ CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, Re
     current = FindCommissioneeDevice(peerAddress);
     if (current != nullptr)
     {
-
-        if (current->IsSecureConnected())
+        if (current->GetDeviceId() == remoteDeviceId)
```

Comment:
In the case where the commissioner is not going to assign an operational ID and operational node ID gets assigned by the signing infrastructure, what will this ID be?  Does the controller have the smarts to select a Temporary Local Node ID as is supposed to be used for this case?  If not, does it select 0x0000_0000_0000_0000 (Unspecified Node ID) and will that still work here?

Reply:
> what will this ID be?

Whatever the API consumer provided when it started commissioning.  This id is basically a handle for the API consumer to identify which commissioning process it's dealing with when it talks to the DeviceCommissioner....